### DISCUSSION TITLE
Hey Everyone

### DISCUSSION THREAD

**Discussion by dufftt** [OpenMined/courses#24] [Introduce Yourself]
Hey Everyone,
I am Sourav Sharma, a junior undergrad student.
 I am very excited to learn federated learning.
 Let's learn and grow together.

**Comment by yemikifouly** [comment]
Hello Sourav! 🙂 You are in for a ride! With the Private AI Series, you won't learn **just** Federated Learning. Instead, you will learn **all** (or most) of the currently relevent technologies for Private AI. So, brace yourself! 🙂 I hope you enjoy this first course very much!
